CHANGELOG.rst

```diff
@@ -20,18 +20,20 @@ Possible types of changes are:
 Unreleased
 ----------
 
-Changed
-'''''''
-- simplify ``op`` decorator
-- simplify default implementation of ``Op.__repr__`
-  Unreleased
-  ----------
-  
+Deprecated
+''''''''''
+- support for unresolved output variables in ``session.evaluate``, use ``sessions.solve`` for that purpose
+- support for arguments of type Variable in ``Op.__call__``, use ``Op.op`` instead
```

Comment:
When decorating a python function `def f(x)` with `@op` can I still run `f(2.0)` to run the decorated function without any reference to a Variable?

Reply:
Yes, and that's all you'll be able to do: when using a variable argument, say ``v``, you'll have to use ``f.op(v)`` from now on.

Reply:
Does the following composition of ops behave as expected?

```python

@op
def f(x):
    return 2. * x


@op
def g(x):
    return f(x) ** 2

v = Variable("v")
x = 2.
y = g(2.)
y
>>> 16. 

# Does this still work
y_v = g.op(v)
evaluate([y_v], {v: 2.})[0]
>>> 16.

```

Reply:
Here, ``f`` will never be used as an op, just as a plain function.